packages/utils/src/browser.ts

```diff
@@ -4,13 +4,18 @@ import { getGlobalObject } from './worldwide';
 // eslint-disable-next-line deprecation/deprecation
 const WINDOW = getGlobalObject<Window>();
 
+const DEFAULT_MAX_STRING_LENGTH = 80;
+
 /**
  * Given a child DOM element, returns a query-selector statement describing that
  * and its ancestors
  * e.g. [HTMLElement] => body > div > input#foo.btn[name=baz]
  * @returns generated DOM path
  */
-export function htmlTreeAsString(elem: unknown, keyAttrs?: string[]): string {
+export function htmlTreeAsString(
```

Comment:
Hmm, actually, this is technically a breaking change (as this method is exported...), so we can't change the signature in that way 😬 

Can you make this:

```ts
export function htmlTreeAsString(
  elem: unknown, 
  keyAttrs?: string[],
  options?: { maxStringLength?: number }
) {}
```

It's not as nice, but will have to do for now!

Or, alternatively, make it a union type:

```ts
export function htmlTreeAsString(
  elem: unknown, 
  options?: { keyAttrs?: string[], maxStringLength?: number } | string[]
) {}
```

And handle this in the code?

Reply:
Agh thanks! I didn't think through how this function isn't merely exposed to other packages within this repo, but that it's also exposed anywhere that imports `@sentry/utils`. I've updated to that latter format, and I've restored the test for the original format.